### PR TITLE
mj-navbar

### PR DESCRIPTION
**Pull request by SebastianStehle**

### CODE REVIEW

File: Mjml.Net/Components/Body/NavbarComponent.cs [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+        [Bind("padding-right", BindType.PixelsOrPercent)]
+        public string? PaddingRight;
+
+        [Bind("padding-top", BindType.PixelsOrPercent)]
+        public string? PaddingTop;
+
+        public override void Render(IHtmlRenderer renderer, GlobalContext context)
+        {
+            context.SetGlobalData("mj-image", new Style(HeadStyle));
```
Comment (LiamRiddell) [inline Mjml.Net/Components/Body/NavbarComponent.cs:77]: Should this be `mj-image`?

**Comment by SebastianStehle** [comment]
mj-navbar is done, but the navbar-links are pending.

**Review by LiamRiddell — CHANGES_REQUESTED** [review]

File: Mjml.Net/Components/Body/NavbarComponent.cs (hunk 1 above)
Comment (SebastianStehle) [reply Mjml.Net/Components/Body/NavbarComponent.cs:77]: True :D

**Review by LiamRiddell — APPROVED** [review]

**Review by LiamRiddell — APPROVED** [review]

**Comment by LiamRiddell** [comment]
It looks really good, nice work :)

**Comment by SebastianStehle** [comment]
It is so easy with all the code generators now.

**Pull request merged**